docs/Configuring-Orion/Using-PostgreSQL.md

```diff
@@ -0,0 +1,57 @@
+description: Storing Information with PostgreSQL 
+<!--- END of page meta data -->
+
+# Store Information with PostgreSQL
+
+PostgreSQL is an object-relational database management system you can use to store key values for Orion.
+
+## Prerequisites
+
+* [PostgreSQL](https://www.postgresql.org/)
+
+## Set up your PostgreSQL database
+
+1. In your database, create the `store` table by using the provided DDL script, as shown in the following example: 
+
+    `psql -h HOST-NAME -U USER-NAME -d DATABASE-NAME -f ./docs/Configuring-Orion/database/postgres_storage.sql` 
```

Comment:
I'm not sure if we have this doc'd in our conventions but we usually use angle brackets around things that need replacing. eg: 
`psql -h <HOST-NAME> -U <USER-NAME> -d <DATABASE-NAME>`

And Bryon and I were talking about this the other day and thought it's probably worth calling it out explicitly when things need replacing to prevent users copying and pasting without replacing them. eg under the example: 
Where `<HOST-NAME>`, `<USER-NAME>`, and `<DATABASE-NAME>` are replaced by the appropriate values for your database.

Reply:
Fixed